cabal.project.ghcjs

```diff
@@ -0,0 +1,7 @@
+compiler: ghcjs
+packages: gauge.cabal
+
+source-repository-package
+  type: git
+  location: https://github.com/ghcjs/ghcjs-base
+  tag: b8d51f65ae1921b2f031710bf75e17f216de442a
```

Comment:
is this file important ? why is a specific tag hardcoded here ?

Reply:
This just got left behind, I will remove this.